forge-gui/res/cardsfolder/upcoming/corrupted_shapeshifter.txt

```diff
@@ -0,0 +1,11 @@
+Name:Corrupted Shapeshifter
+ManaCost:3 U
+Types:Creature Eldrazi Shapeshifter
+PT:*/*
+K:Devoid
+K:ETBReplacement:Other:MorphChoice
+SVar:MorphChoice:DB$ GenericChoice | Defined$ You | Choices$ Flyer,Vigilance,Wall | SpellDescription$ As CARDNAME enters the battlefield, it becomes your choice of a 3/3 creature with flying, a 2/5 creature with vigilance, or a 0/12 creature with defender.
+SVar:Flyer:DB$ Animate | Defined$ Self | Duration$ Permanent | Power$ 3 | Toughness$ 3 | Keywords$ Flying | SpellDescription$ CARDNAME is a 3/3 creature with flying.
+SVar:Vigilance:DB$ Animate | Defined$ Self | Duration$ Permanent | Power$ 2 | Toughness$ 5 | Keywords$ Vigilance | SpellDescription$ CARDNAME is a 2/5 creature with vigilance.
```

Comment:
One weird bug is that only this choice is not working 

![image](https://github.com/Card-Forge/forge/assets/67333662/17fbace8-f6a9-4b11-a443-8eb77ea83d56)

Reply:
haha you used some substitution feature for KW
just give the SVar a different name